code-challenges/fizzBuzzTree/fizz-buzz-tree.js

```diff
@@ -0,0 +1,39 @@
+
+
+const fizzBuzzTree = tree => {
+
+  const collection = [];
+
+  const inOrder = root => {
+    if (root === null) {
+      return;
+    }
+
+    if (root.value % 3 === 0 && root.value % 5 === 0) {
+      root.value = 'FizzBuzz';
+    } else if (root.value % 3 === 0) {
+      root.value = 'Fizz';
+    } else if (root.value % 5 === 0) {
+      root.value = 'Buzz';
+    } else {
+      root.value = root.value.toString();
+    }
```

Comment:
You changed the original tree. Can you modify it to generate a new fizzbuzztree without changing the original tree?